ql/src/test/results/clientpositive/llap/vector_case_when_2.q.out

```diff
@@ -272,49 +272,49 @@ ctimestamp1	ctimestamp2	ctimestamp2_description	ctimestamp2_description_2	ctimes
 1815-05-06 00:12:37.543584705	1815-05-04 22:09:33.543584705	1900s	Old	Old	1815	2018-03-08 23:04:59	12	NULL	1816-05-05
 1883-04-17 04:14:34.647766229	1883-04-16 02:11:30.647766229	1900s	Old	Old	1883	2018-03-08 23:04:59	14	NULL	1884-04-16
 1966-08-16 13:36:50.183618031	1966-08-15 11:33:46.183618031	Early 2010s	Old	Old	1966	1966-08-16 13:36:50.183618031	36	NULL	1967-08-16
-1973-04-17 06:30:38.596784156	1973-04-16 04:27:34.596784156	Early 2010s	Old	Old	1973	1973-04-17 06:30:38.596784156	30	NULL	1973-04-18
+1973-04-17 06:30:38.596784156	1973-04-16 04:27:34.596784156	Early 2010s	Old	Old	1973	1973-04-17 06:30:38.596784156	30	NULL	1974-04-17
```

Comment:
Why these outputs are different from original patch?

Reply:
The commit HIVE-20007 was ported after this commit in OSS master. If we see the diff before that commit in oss master, it is the same as in this PR